department-of-veterans-affairs/va.gov-cms
# VACMS-7022 convert legacy theme code to claro theme

## Description

Relates to #7022 & #6887. Copies existing `vagovadmin` js, scss, and templates to the new `vagovclaro` theme. There are a lot of files in this PR for that reason, but essentially no "new" work. Creating a baseline to refactor from. 

I have commented out many "legacy" styles from being compiled in the new theme, but still included for reference when refactoring. The only template change so far is updated the legacy `fieldset.html.twig` override based on seven to the `fieldset.html.twig` from claro.

## Testing done

- Verified that theme libraries attach to expected pages, helper tooltips & centralized content still behave as expected.
- Verified that the build process correctly transpiles es6 javascript code in the new theme.
- Verified that the gulp process correctly converts sass to css in the new theme.
- Verified that none of the forms look _too wonky_, there are definitely places to clean up. Will be creating tickets today to track these tasks.

## Screenshots
<img width="942" alt="Screen Shot 2021-11-24 at 10 51 07 AM" src="https://user-images.githubusercontent.com/11279744/143299498-2f565ebb-1222-442f-a43b-6461ac4460cb.png">

<img width="1593" alt="Screen Shot 2021-11-24 at 11 20 50 AM" src="https://user-images.githubusercontent.com/11279744/143300904-49c92dff-cf78-4267-91a4-2acf51597009.png">

## QA steps

As user with admin role,
1. enable vagovclaro as admin theme
2. click around various editor forms and pages, see that no functionality should be changed just different visual styles.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?

No announcement for Claro work will be needed until we get closer to flipping the switch for all editors/a pilot subset.
